src/test/java/in/ashwanthkumar/gocd/artifacts/JanitorTest.java

```diff
@@ -25,55 +26,71 @@
 import static org.mockito.Mockito.when;
 
 public class JanitorTest {
-    Janitor janitor = new Janitor(mock(Action.class));
+    Action action = mock(Action.class);
+    GoCD client;
+    JanitorConfiguration config = new JanitorConfiguration();
+
+    @Before
+    public void before() {
+        client = mock(GoCD.class);
+        config = new JanitorConfiguration();
+    }
+
+    private Janitor janitor() {
```

Comment:
You can just push the changes in your same branch - it would automatically update in this PR. Just drop me a note once it's done.